ansible/roles/ansible-tvault-contego-extension/tasks/contego_service.yml

```diff
@@ -57,22 +57,3 @@
      src: contego_service_debian.j2
      dest: "{{contego_service_debian_path}}"
   when: ansible_distribution=="Ubuntu" and (ansible_lsb.codename != xenial and ansible_lsb.codename != willy and ansible_lsb.codename != bionic)
-
-- name: enable and daemon reload tvault contego service 
-  block:
-    - shell: systemctl daemon-reload
-    - name: enable tvault-contego service and reload it
-      service: name=tvault-contego  state=restarted enabled=yes
-  when: >
-       (ansible_distribution_major_version=="7" and ansible_distribution== centos) or
-       (ansible_distribution_major_version=="7" and ansible_distribution=="RedHat") or
-       (ansible_distribution== suse) or (ansible_distribution== SLES) or
-       (ansible_distribution=="Ubuntu" and (ansible_lsb.codename == xenial or ansible_lsb.codename == willy or ansible_lsb.codename == bionic))
-
-- name: restart tvault-contego
-  service: name=tvault-contego state=restarted
-  when: >
-       (ansible_distribution_major_version=="7" and ansible_distribution== centos) or
-       (ansible_distribution_major_version=="7" and ansible_distribution=="RedHat") or
-       (ansible_distribution== suse) or (ansible_distribution== SLES) or
-       (ansible_distribution=="Ubuntu" and (ansible_lsb.codename == xenial or ansible_lsb.codename == willy or ansible_lsb.codename == bionic))
```

Comment:
why are we moving this section to object store ? In case of nfs, we don't need object store.

Reply:
Without this moving tvault contego service is failing 
After moving this changes 
We tested 3 times uninstallation or installation
Its working fine